test/e2e/federated-ingress.go

```diff
@@ -58,8 +61,56 @@ const (
 	FederatedIngressGCLBSrcRange = "130.211.0.0/22"
 
 	// TLS Certificate and Key for the ingress resource
```

Comment:
Add instructions on how you generated them to make it easier for the guy who wants to change these